src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -283,6 +283,6 @@ export default {
 	.dynamic-hero-classic::v-deep i,
 	.dynamic-hero-classic::v-deep b,
 	.dynamic-hero-classic::v-deep u {
-		color: rgb(var(--text-action));
+		@apply tw-text-brand-300;
```

Comment:
Are none of the theme vars suitable for this? https://github.com/kiva/kv-ui-elements/blob/main/%40kiva/kv-tokens/primitives.json#L74

If not please create a ticket to update the themes and to use a theme variable. Otherwise, this will break if a different theme is chosen.